src/main/kotlin/baseball/controller/GameController.kt

```diff
@@ -0,0 +1,37 @@
+package baseball.controller
+
+import baseball.domain.Computer
+import baseball.domain.GameResult
+import baseball.service.GameService
+import baseball.view.InputView
+import baseball.view.OutputView
+
+class GameController(
+        private val inputView: InputView,
+        private val outputView: OutputView,
+        private val gameService: GameService
+) {
+
+    fun run() {
+        outputView.printStart()
+
+        var gameCommand: GameCommand = GameCommand.play()
```

Comment:
강력한 타입추론을 많이 활용해도 좋아요..!
```suggestion
        var gameCommand = GameCommand.play()
```

Reply:
play 했는데 어떤 타입이 튀어나올지 다른 개발자가 몰라볼까봐 남겼어요ㅋㅋ